keras/layers/activation/relu.py

```diff
@@ -65,9 +65,9 @@ class ReLU(Layer):
       Same shape as the input.
 
     Args:
-      max_value: Float >= 0. Maximum activation value. Default to None, which
-        means unlimited.
-      negative_slope: Float >= 0. Negative slope coefficient. Default to 0.
+      max_value: Float >= 0. Maximum activation value. None
+        means unlimited. Defaults to `None`.
```

Comment:
Please move the "means unlimited" to the previous line.

Reply:
What should it be replaced with? `max_value` of `None` doesn't seem to make sense and needs a docstring to explain it. Is it `sys.float_info.max`? - Or `float("inf")`?